seed/seed-db.js

```diff
@@ -2,26 +2,33 @@
 /* eslint-disable import/first */
 process.env.DEBUG = 'mongo-seeding'; // https://www.npmjs.com/package/mongo-seeding#debug-output
 const { Seeder } = require('mongo-seeding');
-const path = require('path');
-require('./config');
+const yamlConfig = require('node-yaml-config');
 
-const collectionsPath = path.resolve(__dirname, 'collections');
+const path = require('path');
+const { AWS_PROFILE } = require('../config');
+require('../config');
 
-const config = {
-  database: process.env.MONGODB_URI,
-  dropDatabase: true,
-  dropCollections: true,
-};
+const dataPath = path.resolve(__dirname, 'data');
 
 function seedDb() {
+  process.env.AWS_PROFILE = AWS_PROFILE;
+  const envConfig = yamlConfig.load(path.resolve(__dirname, '../env.yml'));
+  const config = {
+    database: envConfig.MONGODB_URI,
+    dropDatabase: true,
```

Comment:
We should not be dropping the database. There might be other collections that are not part of the seeded data that we don't want to get rid of.